jiant/preprocess.py

```diff
@@ -235,7 +235,11 @@ def _build_vocab(args, tasks, vocab_path: str):
 
 def build_indexers(args):
     indexers = {}
-    if not args.input_module.startswith("bert") and args.input_module not in ["elmo", "gpt"]:
+    if (
+        not args.input_module.startswith("bert")
```

Comment:
There seems to be a few places where we have this growing list of contextualized word embedding models that behave differently than glove/cove/RNN based ones - perhaps find a way to move this list to a more central location so we avoid adding to this list every time we add a new type of contextualized word embedding model?

Reply:
Many of the lists differ slightly, and most of them require this startswith logic that's hard to convert into a list constant. Open to specific suggestions, but I don't see anything easy here.

Reply:
Third option for this block specifically: just make it `args.input_module in ['scratch', 'glove', 'fasttext']` ?
We know exactly what should fall in this bucket.